script.skinvariables/addon.xml

```diff
@@ -1,21 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon id="script.skinvariables"
-version="0.0.3"
+version="0.2.5"
 name="Skin Variables"
 provider-name="jurialmunkey">
 <requires>
   <import addon="xbmc.python" version="3.0.0"/>
 </requires>
 <extension point="xbmc.python.script" library="script.py" />
 <extension point="xbmc.addon.metadata">
-  <summary lang="en_GB">Skin Variables helps skinners to construct variables for multiple containers and listitems from a template</summary>
-  <description lang="en_GB">Skin Variables helps skinners to construct variables for multiple containers and listitems from a template</description>
+  <summary lang="en_GB">Skin Variables helps skinners to construct variables and expressions for multiple containers and listitems using a template</summary>
+  <description lang="en_GB">Skin Variables helps skinners to construct variables and expressions for multiple containers and listitems using a template</description>
   <disclaimer lang="en_GB">For skinners</disclaimer>
-  <license>GPL-3.0-or-later</license>
+  <license>GPL-3.0</license>
```

Comment:
```suggestion
  <license>GPL-3.0-only</license>
```

Reply:
Strange, that suffix isn't in my repo's commit history at all - I must've added it to an earlier PR on code review but forgot to push the change back to mine. Anyway, reverted back to the `-or-later` license so it stays consistent (and also updated on my repo so that the suffix won't go missing again).